pkg/report/linux.go

```diff
@@ -1883,7 +1883,9 @@ var linuxOopses = append([]*oops{
 				},
 			},
 		},
-		[]*regexp.Regexp{},
+		[]*regexp.Regexp{
+			compile("general protection fault .* in syz-executor"),
```

Comment:
This is also applicable to other processes. I think our C reproducers are named differently. But also syz-fuzzer can crash, or just any binary in the distro.

Reply:
Repros are also prefixed as syz-executor (in fact, the test is based on the repro output), but yeah, definitely makes sense.

Updated. PTAL.